store.go

```diff
@@ -3187,9 +3188,15 @@ func stringSliceWithoutValue(slice []string, value string) []string {
 	modified := make([]string, 0, len(slice))
 	for _, v := range slice {
 		if v == value {
-			continue
+			ref, err := reference.ParseNamed(v)
+			if err != nil {
+				continue
+			}
+			name := fmt.Sprintf("%s:<none>", reference.TrimNamed(ref).String())
```

Comment:
`<none>` is not a valid tag (as is already visible from the fact that this needs to use a `Sprintf` instead of `reference.WithTag`).

While it’s not, IIRC, documented explicitly, c/storage names must _all_ be parseable using `ParseNormalizedNamed` (tools like podman just fail when they see a non-compliant name).